### ISSUE TITLE
unimported fixture used in script: tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py

### ISSUE THREAD

**Issue by rraghav-cisco** [sonic-net/sonic-mgmt#14333]
https://github.com/sonic-net/sonic-mgmt/blob/d521d1e71f26fbd1c3b298ab086a490175e5b60c/tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py#L105

This script uses fanout_graph_facts and fanout_graph_facts_multidut(which I presume is the correct one). But imports only the multidut variant. Pls fix the script.

The error is below:
```
                "summary": "failed on setup with \"file /data/tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py, line 273
  @pytest.mark.disable_loganalyzer
  @pytest.mark.parametrize('reboot_type', ['cold'])
  #@pytest.mark.parametrize('reboot_type', ['warm', 'cold', 'fast'])
  @pytest.mark.parametrize(\"multidut_port_info\", MULTIDUT_PORT_INFO[MULTIDUT_TESTBED])
  def test_pfc_pause_multi_lossy_prio_reboot(snappi_api,          # noqa: F811
E       fixture 'fanout_graph_facts' not found
>       available fixtures: __pytest_repeat_step_number, active_active_ports, active_active_ports_config, active_standby_ports, add_mgmt_test_mark, all_prio_list, ansible_adhoc, ansible_facts, ansible_module, asic_db_dut, asic_db_dut_rand, asic_db_dut_supervisor, backup_and_restore_config_db_session, cable_type, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, check_bgp, check_dbmemory, check_dut_asic_type, check_interfaces, check_ipv6_mgmt, check_monit, check_mux_simulator, check_neighbor_macsec_empty, check_processes, c",
```

**Comment by rraghav-cisco** [comment]
@sdszhang , @selldinesh , @abdosi , @kamalsahu0001 : FYI.

**Comment by sdszhang** [comment]
Fixed in #14742
